libs/nox/src/spatial.rs

```diff
@@ -142,6 +149,7 @@ impl<T: TensorItem + Field + NativeType + ArrayElement> SpatialInertia<T> {
         SpatialInertia { inner }
     }
 
+    /// Constructs spatial inertia from a mass, assuming inertia and momentum are derived from it.
```

Comment:
This assumes momentum is 0 and the inertia is the same value as the mass along all axes.